interface/modules/custom_modules/oe-module-comlink-telehealth/public/assets/js/src/registration-checker.js

```diff
@@ -36,7 +36,7 @@ export function RegistrationChecker(scriptLocation)
                 return result.json();
             })
             .then(registrationSettings => {
-                if (registrationSettings && registrationSettings.hasOwnProperty('errorCode')) {
+                if (registrationSettings && Object.prototype.hasOwnProperty.call(registrationSettings, 'errorCode')) {
```

Comment:
If you are going to change these values you need to rebuild the minified json in the telehealth src using 
```
cd ./openemr/interface/modules/custom_modules/oe-module-comlink-telehealth/public/assets/js/
npm run build
```